src/test/java/edu/cornell/kfs/module/purap/jaggaer/supplier/xml/JaggaerExampleTest.java

```diff
@@ -430,14 +430,14 @@ private DiversityClassificationList buildDiversityClassificationList() {
         
         AdditionalDataList dataList = new AdditionalDataList();
         
-        AdditionalData dataum = new AdditionalData();
+        AdditionalDataItem dataum = new AdditionalDataItem();
```

Comment:
what does the variable name "dataum" mean?

Reply:
looks like this was probably a typo on my part, I will fix it.  This should be datum, which is the singular form data

Reply:
I know that this has already been approved, merged, and probably promoted to production, but I just saw this today, and I want to jump in. 

Using "datum" is fraught with opinion. As far as I've noticed in the past 20 years, I've never heard anyone say, "those data were written to the log file." In most situations, "data" is commonly used as a mass noun, like "information", to refer to a collection of more than one datum, and a collection is singular. Furthermore the number of times I have actually heard someone use the word "datum" has declined to almost zero.

In the case of this code, the class is called "AdditionalDataItem", so for consistency, the least confusing variable name would be "dataItem", not "datum", like what was used on line 431 with "AdditionalDataList dataList = ...".

Just sticking my nose in where it doesn't belong...